src/cart/crt_swim.c

```diff
@@ -206,31 +262,44 @@ static void crt_swim_cli_cb(const struct crt_cb_info *cb_info)
 	crt_rpc_t		*rpc_req = cb_info->cci_rpc;
 	struct crt_rpc_swim_in	*rpc_swim_input = crt_req_get(rpc_req);
 	struct swim_member_state id_state;
+	swim_id_t		 self_id = swim_self_get(ctx);
 	swim_id_t		 id = rpc_req->cr_ep.ep_rank;
-	int			 rc;
+	int			 rc = 0;
 
 	D_TRACE_DEBUG(DB_TRACE, rpc_req,
 		      "complete opc %#x with %zu updates %lu => %lu "DF_RC"\n",
 		      rpc_req->cr_opc, rpc_swim_input->upds.ca_count,
 		      rpc_swim_input->src, id, DP_RC(cb_info->cci_rc));
 
+	if ((rpc_req->cr_opc & 0xFFFF) == 0) /* is one way RPC? */
+		D_GOTO(out, rc);
```

Comment:
I am not sure how you determine one way rpc via opcode like this; i couldnt find place on code where we change opcode based on whether it is one way rpc or not.

Should instead check for: 
rpc_priv->crp_opc_info->coi_no_reply

Reply:
Hmm. Good ctach! I will loot at this and replace if it will work properly.